<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #474953;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="applicationTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>1211</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E1E1E;</string>
    </property>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTabWidget" name="tabDisplayWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>500</width>
        <height>528</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="graphWidget">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="paintWidget">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QWidget" name="informationWIdget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>550</y>
        <width>211</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget
{
	border: 1px solid black;
}</string>
      </property>
      <widget class="QLabel" name="labelTotalTime">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Total time</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelAccuracy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Accuracy</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFMeasure">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>F-measure</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelRecall">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Recall</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPrecision">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Precision</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>30</y>
        <width>271</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget
{
	border: 1px solid black;
}</string>
      </property>
      <widget class="QLabel" name="labelModelSelection">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="chooseMatrixModel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Graph model</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="chooseGraphModel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Matrix model</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNumberOfHiddenLayers">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Hidden layers</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="hiddenLayers">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>44</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox
{
	color: #C8C8C8;
	background-color: #40424B;
	border: 1px solid #393C46;
	border-radius: 4;
}</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
      <widget class="QPushButton" name="selectGraph">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>594</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:  #40424B;
	color: #B4B4B4;
	border: solid;
	border-radius: 6.5;
}</string>
       </property>
       <property name="text">
        <string>Graph</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectPaint">
       <property name="geometry">
        <rect>
         <x>138</x>
         <y>594</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:  #40424B;
	color: #B4B4B4;
	border: solid;
	border-radius: 6.5;
}</string>
       </property>
       <property name="text">
        <string>Paint</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTraining">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Training</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNumberOfEpochs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Number of epochs</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNumberOfGroups">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Number of groups</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="numberOfEpochs">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>230</y>
         <width>44</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox
{
	color: #C8C8C8;
	background-color: #40424B;
	border: 1px solid #393C46;
	border-radius: 4;
}</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="numberOfGroups">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>44</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox
{
	color: #C8C8C8;
	background-color: #40424B;
	border: 1px solid #393C46;
	border-radius: 4;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRealTime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:  #40424B;
	color: #B4B4B4;
	border: solid;
	border-radius: 6.5;
}</string>
       </property>
       <property name="text">
        <string>Start the learning process in real time</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCrossValidation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:  #40424B;
	color: #B4B4B4;
	border: solid;
	border-radius: 6.5;
}</string>
       </property>
       <property name="text">
        <string>Start the learning process with cross-validation</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openSettings">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>530</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color:  #40424B;
	color: #B4B4B4;
	border: solid;
	border-radius: 6.5;
}</string>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLearningRateMain">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>Learning rate</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="learningRate">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>65</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
	color: #C8C8C8;
	background-color: #40424B;
	border: 1px solid #393C46;
	border-radius: 4;
}</string>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="exitSettings">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>650</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color:  #40424B;
	color: #B4B4B4;
	border: solid;
	border-radius: 6.5;
}</string>
      </property>
      <property name="text">
       <string>Exit settings</string>
      </property>
     </widget>
     <widget class="QWidget" name="activationWidget" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>590</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget::hover
{
	border: 0.5px solid #6495ED;
}</string>
      </property>
      <widget class="QLabel" name="labelActivationFunction">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Activation function</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelActivationFunction_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>461</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QComboBox" name="activationFunction">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox 
{
	background-color: #3C3C3C;
	color: #AAAAAA;
	border-radius: 4px;
	padding-left: 10px;
	font-weight: bold;
}

QComboBox::drop-down
{
	border: 0px;
}

QComboBox::down-arrow
{
	
	image: url(:/resources/images/Icons8_flat_expand.svg.png);
	width: 12px;
	height: 12px;
	margin-right: 15px;
}

QComboBox::hover
{
	border: 0.5px solid #CED4DA;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Sigmoid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SiLU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ReLU</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="weightWidget" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>590</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget::hover
{
	border: 0.5px solid #6495ED;
}</string>
      </property>
      <widget class="QLabel" name="labelWeight">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelWeight_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>461</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QComboBox" name="weightFunction">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox 
{
	background-color: #3C3C3C;
	color: #AAAAAA;
	border-radius: 4px;
	padding-left: 10px;
	font-weight: bold;
}

QComboBox::drop-down
{
	border: 0px;
}

QComboBox::down-arrow
{
	
	image: url(:/resources/images/Icons8_flat_expand.svg.png);
	width: 12px;
	height: 12px;
	margin-right: 15px;
}

QComboBox::hover
{
	border: 0.5px solid #CED4DA;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Xavier</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="momentumWidget" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>590</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget::hover
{
	border: 0.5px solid #6495ED;
}</string>
      </property>
      <widget class="QLabel" name="labelMomentum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Momentum</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMomentum_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>461</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="momentum">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
	background-color: #3C3C3C;
	color: #AAAAAA;
	border-radius: 4px;
	padding-left: 10px;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="momentumWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>380</y>
        <width>590</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget::hover
{
	border: 0.5px solid #6495ED;
}</string>
      </property>
      <widget class="QLabel" name="labelLearningRate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Learning rate reduction ratio</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLearningRate_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>461</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
}</string>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="learningRateEpochK">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
	background-color: #3C3C3C;
	color: #AAAAAA;
	border-radius: 4px;
	padding-left: 10px;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>0.75</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="learningRateLayerK">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
	background-color: #3C3C3C;
	color: #AAAAAA;
	border-radius: 4px;
	padding-left: 10px;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>0.75</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLearningRateEpochs">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Epochs</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLearningRateLayers">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	border: 0px;
	color: #AAAAAA;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Layers</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFIle"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFIle">
   <property name="text">
    <string>Open file...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
